website/mailing_list/model.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+from modularodm import fields, StoredObject
+
+from framework.mongo import ObjectId
+from framework.mongo.utils import unique_on
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    date_created = fields.DateTimeField(default=datetime.datetime.utcnow, index=True)
```

Comment:
Only index fields that you plan on querying on. If you are not querying on these fields alone it's a wasted index.
And a note for dates. You generally want to index them in descending order rather than ascending, that way the latest dates will be the first items in the index, see `NodeLog` for an example.